src/libAtomVM/smp.h

```diff
@@ -223,7 +223,7 @@ int smp_get_online_processors();
 /**
  * @brief Start a new scheduler, calling `scheduler_entry_point` with the given
  * global context.
- * @param glb the global context
+ * @param ctx the global context
  */
 void smp_scheduler_start(GlobalContext *ctx);
```

Comment:
A bug in the code is worth two in the documentation, yet I'd change the parameter name in the signature here, we really want `glb`.